### ISSUE TITLE
How to specify index sort order of each column?

### ISSUE THREAD

**Issue by jjxtra** [jsakamoto/EntityFrameworkCore.IndexAttribute#1]
Is it possible to add another property of the Index attribute to specify the index sort order for each column (ASC / DESC)?

**Comment by jsakamoto** [comment]
No, it is impossible to specify the sort order (ASC/DESC) in the IndexAttributes at this time, because EntityFramework Core doesn't expose the feature of sort order control.

Anyway, this issue is a good feature request, I think.

But I should spend a lot of a time  to learning the way of customizing SQL script generator in EFCore for realizing this feature.

See also: https://romiller.com/2012/01/16/customizing-code-first-migrations-provider/  
(The topic above isn't about EF Core, it is about classic EF. But the topic is still useful.)

**Comment by alikleitcr7** [comment]
Any update on this? 

This is a rather important feature that should be included in EF Core which is essential for the query's performance without having to add a custom code migration provider. Customizing the code on future bases can break at some point in future releases.
